### PR TITLE
fix: fix method and value accessors for SSR

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: src/components/radio-button/radio-button-group/radio-button-group.ts [hunk 1]
```diff
@@ -232,9 +232,11 @@ export class SbbRadioButtonGroup extends LitElement {
       return;
     }
 
-    this.value = initValue ?? this._radioButtons.find((radio) => radio.checked)?.value;
+    const radioButtons = this._radioButtons;
```
Comment (kyubisation) [inline src/components/radio-button/radio-button-group/radio-button-group.ts:235]: Purpose of this variable? Performance impact should be negligible?

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
A few comments to discuss/consider

File: src/components/radio-button/radio-button-group/radio-button-group.ts (hunk 1 above)
Comment (jeripeierSBB) [reply src/components/radio-button/radio-button-group/radio-button-group.ts:235]: Did expect to make you happy increasing performance 😃

Comment (jeripeierSBB) [reply src/components/radio-button/radio-button-group/radio-button-group.ts:235]: It‘s a getter which each time does a queryselectorall

Comment (kyubisation) [reply src/components/radio-button/radio-button-group/radio-button-group.ts:235]: > Did expect to make you happy increasing performance 😃

More code vs better performance is always a trade-off 😄 
But this is fine then 👍

File: src/components/image/image.ts [hunk 2]
```diff
@@ -260,11 +260,11 @@ export class SbbImage extends LitElement {
   }
 
   private _prepareImageUrl(baseUrl: string, lquip = false): string {
-    if (!baseUrl || baseUrl === '') {
+    if (!baseUrl || baseUrl === '' || !isBrowser()) {
       return '';
     }
 
-    const imageUrlObj = new URL(baseUrl);
+    const imageUrlObj = typeof URL !== 'undefined' && new URL(baseUrl);
```
[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/components/image/image.ts:267]: It showed me in lib.dom.d.ts file. but tested in node now and it's available as you said :)

File: src/components/navigation/navigation-marker/navigation-marker.ts [hunk 3]
```diff
@@ -89,16 +89,16 @@ export class SbbNavigationMarker extends LitElement {
...
-    this._actions = Array.from(this.children).filter(
+    this._actions = Array.from(this.children ?? []).filter(
       (e): e is SbbNavigationAction => e.tagName === 'SBB-NAVIGATION-ACTION',
     );
   }
 
   private _setMarkerPosition(): void {
     if (this._hasActiveAction) {
-      this?.style.setProperty(
+      this.style.setProperty(
```
[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/components/navigation/navigation-marker/navigation-marker.ts:99]: I thought "this" is always defined. But the style should have a ?. Fixed in new commit

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Pull request merged**